packages/cli/tests/unit/rpc/navie/models/registry.spec.ts

```diff
@@ -0,0 +1,212 @@
+import ModelRegistry from '../../../../../src/rpc/navie/models/registry';
+
+describe(ModelRegistry, () => {
+  beforeEach(() => {
+    (ModelRegistry as any).instance = new (ModelRegistry as any)();
+  });
```

Comment:
That's naughty ;) Is there a reason not to just use a `registry = new ModelRegistry()` in the tests?